fe/fe-core/src/main/java/com/starrocks/qe/HDFSBackendSelector.java

```diff
@@ -272,6 +278,13 @@ public void computeScanRangeAssignment() throws UserException {
             }
             recordScanRangeAssignment(node, scanRangeLocations);
         }
+
+        // record statistic
```

Comment:
please record how many scan range are assigned because of rebalance instead of affinity